task2_coding_exercises/q1_arrays.py

```diff
@@ -49,4 +63,20 @@ def most_freq_element(arr: list[int]) -> int:
     """
 
     # insert code here
-    pass
+    count = {}
+
+    #store how many of each number appears in a dictionary
+    for i in range(len(arr)):
+        if arr[i] in count:
+            count[arr[i]] += 1
+        else:
+            count[arr[i]] = 1
+        
+        max = 0
```

Comment:
wrong indentation?

Reply:
Sorry I meant to write the function like 


```
def most_freq_element(arr: list[int]) -> int:

    # insert code here
    count = {}
    #store how many of each number appears in a dictionary
    for i in range(len(arr)):
        if arr[i] in count:
            count[arr[i]] += 1
        else:
            count[arr[i]] = 1
            
    max = 0
    maxList = []
    for key in count:
        if count[key] > max:
            max = count[key]
            maxList.append(key)

    return maxList[0]
```